frontend/src/main/scala/bloop/Project.scala

```diff
@@ -22,6 +22,8 @@ case class Project(name: String,
                    javaEnv: JavaEnv,
                    tmp: AbsolutePath,
                    bloopConfigDir: AbsolutePath) {
+  override def toString: String = s"$name"
```

Comment:
`name` ?

Reply:
Yes, otherwise the pretty-printed version of the project is terrible: it prints all the malformatted arrays to the users. This fixes some usability errors.
  
In the future, after 1.0.0, I want to use `Show` for practically everything. But now that we're under a deadline, I just prefer to do it this way.